### PR TITLE
port force-mode-line-update

### PR DESCRIPTION
**Pull request by brotzeit**

### CODE REVIEW

File: rust_src/src/buffers.rs [hunk 1]
```diff
@@ -799,6 +800,28 @@ pub fn buffer_base_buffer(buffer: LispObject) -> Option<LispBufferRef> {
...
+/// header lines.  This function also forces recomputation of the
+/// menu bar menus and the frame title.
+#[lisp_fn(min = "0")]
+pub fn force_mode_line_update(all: LispObject) -> LispObject {
+    let mut current_buffer = ThreadState::current_buffer();
+    if all.is_not_nil() {
+        unsafe {
+            update_mode_lines = 10;
+        }
+        // FIXME: This can't be right.
```
[the remark this answers is not in the document]
Comment (brotzeit) [reply rust_src/src/buffers.rs]: If I would have any idea what needs to be fixed here :P

[the remark this answers is not in the document]
Comment (shaleh) [reply rust_src/src/buffers.rs]: https://github.com/emacs-mirror/emacs/commit/655ab9a380068143cfb9a31d01488e83676d81c1

This is the commit where the comment was added.

[the remark this answers is not in the document]
Comment (db48x) [reply rust_src/src/buffers.rs]: Sorry, I was being facetious and didn't mean to hold this patch up :)

[the remark this answers is not in the document]
Comment (brotzeit) [reply rust_src/src/buffers.rs]: I wasn't sure if you were serious ;)

**Pull request merged**